.github/workflows/build-release.yml

```diff
@@ -83,11 +86,17 @@ jobs:
         with:
           profile: minimal
 
+      # https://github.com/actions/virtual-environments/issues/2557#issuecomment-769611326
+      - if: ${{ matrix.target }} == 'aarch-apple-darwin'
+        run: |
+          sudo xcode-select -s /Applications/Xcode_12.4.app &&
```

Comment:
Is that `&&` mandatory?

Reply:
Yes, it is. Because github action runner will report the status code of the last line, we need to make sure this is executed as a single line.

[ref](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsrun)
> sh-like shells exit with the exit code of the last command executed in a script, which is also the default behavior for actions. The runner will report the status of the step as fail/succeed based on this exit code.